### DISCUSSION TITLE
Link Select using startSelecting

### DISCUSSION THREAD

**Discussion by reshin-pulapaddy** [clientIO/joint#2290] [Q&A]
### Introduction

I am using `selection.startSelecting(evt) `to drag and select the canvas objects, but it only selects the elements
Is it possible to select links as well?
![drag select](https://github.com/clientIO/joint/assets/117301232/614b2917-651c-4f98-a7e1-a9201cdbb445)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
Duplicate of https://github.com/clientIO/joint/discussions/2024.
